terraform/modules/benchmark/service/bench.rb

```diff
@@ -270,10 +267,12 @@ def run_benchmarks(benchmarks, options)
   Dir.chdir(WORKING_DIR) do
     benchmarks_loop(benchmarks, options, bench_metrics)
   end
-  if options[:daily]
-    write_csv(bench_metrics, options)
-  else
-    write_markdown(bench_metrics)
+  handle_exception(nil) do
```

Comment:
Using `false` instead of `nil` could be a bit clearer.

Reply:
Or even `@db_created = false`.

Reply:
Updated.